noxfile.py

```diff
@@ -322,6 +322,7 @@ def mypy(session: nox.Session) -> None:
         version=session.python,
         resolution="highest",
     )
+    session.install("pip")
```

Comment:
For some reason I was getting an error that `pip` was not installed, at least when running `nox -s mypy` locally. Maybe because I was using a `uv` backend?